rpfs/models.py

```diff
@@ -0,0 +1,95 @@
+# Here was one attempt but I went back and did the others below it once we spoke. Still sending you my initial confusion fort clarity
+
+from django.db import models
+from datetime import datetime
+
+
+class ProductCategory(models.Model):
+    product_category = models.Charfield(max_Length=200)
+    category_summary = models.Textfield(max_Length=500)
+    category_slug = models.Charfield(max_Length=200)
+
+    class Meta:
+        verbose_name_plural = "Categories"
+
+        def __str__)
+
+            self:
+        return self.product_category
+
+
+class ProductDescription(model.Models):
+    product_description = models.TextField(max_Length=500)
+    product_category = models.ForeignKey(ProductCategory, default=1, verbose_name="Category", on
+    delete = models.SET_DEFAULT)
+    product_title = models.CharField(max_Length=100)
+
+    class Meta:
+        verbose_name_plural = "Title"
+
+        def __str__)
+
+            self:
+        return self.product_title
+
+
+# Should this actually be labled "Product Description"? Or flip (or delete) the code for "Product Title" with "Project Description"?
```

Comment:
I would combine ProductTitle and ProductDescription.  A good rule about when to make something a separate model is to ask whether an item might have more than one.  We know that every category will relate to multiple products, but each product is probably going to have just one title and description.